test/functional/JIT_Test/src/jit/test/recognizedMethod/TestJavaLangMath.java

```diff
@@ -55,4 +62,86 @@ public void test_java_lang_Math_sqrt() {
         AssertJUnit.assertEquals(Double.POSITIVE_INFINITY, Math.sqrt(Double.POSITIVE_INFINITY));
         AssertJUnit.assertTrue(Double.isNaN(Math.sqrt(Double.NaN)));
     }
+
+    @Test(groups = {"level.sanity"}, invocationCount=2, dataProvider="zeroProvider", dataProviderClass=TestMathUtils.class)
```

Comment:
As these methods are only testing floating point types max / min operations, we should reflect that in a name that is more clear. May be you can use postfix `_FD` which is the JVM descriptor for floating point float and doubles in all test methods?